doc/source/coregistration.md

```diff
@@ -476,6 +550,8 @@ For both **inputs** and **outputs**, four consistent categories of metadata are
 
 **4. Affine metadata (common to all affine methods)**:
 
+- An input `only_translation` to define if a coregistration should solve only for translations,
+- An input `standardize` to define if the input data should be standardized to the unit sphere before coregistration,
```

Comment:
This is not super clear. Is it better described somewhere, that you could point to?

Reply:
Tried to clarify as well.